### PR TITLE
Fixing the build of images

### PR DESCRIPTION
**Pull request by tpapagian**
Signed-off-by: Anastasios Papagiannis <tasos.papagiannnis@gmail.com>

### CODE REVIEW

File: install/kubernetes/values.yaml
Comment (joestringer) [inline install/kubernetes/values.yaml:55]: This makes sense if the values in the `main` branch are pointing to latest, since we have the release gating in place to ensure that `quay.io/cilium/tetragon` requires maintainer signoff to push new images there (ie only for full official releases). FWIW in Cilium, for release branches and tags we then go and rewrite the `tag` below to the relevant tag when we release the helm charts. At the same time we would do the equivalent of ensuring that this repository gets switched back to the main repo like `quay.io/cilium/tetragon`.

Probably just need to integrate this into whatever release scripting we set up.

**Review by joestringer — APPROVED** [review]
Good to see that the quay access rights are properly restricting the image pushes.

**Review by michi-covalent — APPROVED** [review]

**Pull request merged**
